generic-database-plugin/src/test/java/io/cdap/plugin/db/batch/sink/DBSinkTest.java

```diff
@@ -1,149 +0,0 @@
-/*
- * Copyright © 2019 Cask Data, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not
- * use this file except in compliance with the License. You may obtain a copy of
- * the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations under
- * the License.
- */
-
-package io.cdap.plugin.db.batch.sink;
-
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.Maps;
-import io.cdap.cdap.api.data.format.StructuredRecord;
-import io.cdap.cdap.api.data.schema.Schema;
-import io.cdap.cdap.api.dataset.lib.KeyValue;
-import io.cdap.cdap.etl.mock.common.MockEmitter;
-import io.cdap.plugin.db.DBRecord;
-import io.cdap.plugin.jdbc.DatabaseSink;
-import org.apache.hadoop.io.NullWritable;
-import org.junit.Assert;
-import org.junit.Test;
-
-import java.util.Map;
-import java.util.stream.Collectors;
-
-/**
- * Tests for {@link AbstractDBSink}
- */
-public class DBSinkTest {
```

Comment:
why remove this?

Reply:
This class tested transform method in Sink plugin. For now transform method doesn't validate schema from db and input. Instead of this class I’ve added unit tests, which cover these cases, in CommonFieldsValidatorTest class in database-commons module.